beaker/services/service_client.py

```diff
@@ -84,16 +85,33 @@ def make_request(session: requests.Session) -> requests.Response:
             try:
                 response.raise_for_status()
             except requests.exceptions.HTTPError:
+                # Try parsing error message from the response
+                msg: Optional[str] = None
                 if response.text:
                     try:
                         msg = json.loads(response.text)["message"]
-                        raise BeakerError(msg)
                     except (TypeError, KeyError, json.JSONDecodeError):
                         pass
-                raise
+
+                if (
+                    msg is not None
+                    and response.status_code is not None
+                    and 400 <= response.status_code < 500
```

Comment:
429 is not in this. But I guess beaker doesn't report that status anyways, so it's fine?